jf_agent/agent_logging.py

```diff
@@ -125,29 +166,22 @@ def log_loop_iters(
 }
 
 
-def log_and_print(logger, level, msg):
-    '''
-    For an info-level message that should be sent to the logger, and also written
-    to stdout (for user visibility)
-    '''
-    logger.log(level, msg)
-    print(msg, flush=True)
+def generate_standard_error_msg(error_code, msg_args=[]):
+    return f'[{error_code}] {ERROR_MESSAGES.get(error_code).format(*msg_args)}'
 
 
-def log_and_print_error_or_warning(logger, level, error_code, msg_args=[], exc_info=False):
+def log_standard_error(logger, level, error_code, msg_args=[], exc_info=False):
     '''
     For a failure that should be sent to the logger with an error_code, and also written
     to stdout (for user visibility)
     '''
     assert level >= logging.WARNING
-    msg = f'[{error_code}] {ERROR_MESSAGES.get(error_code).format(*msg_args)}'
-    logger.log(level, msg, exc_info=exc_info)
-    print(msg, flush=True)
-    if exc_info:
-        print(traceback.format_exc())
-
-
-# TODO(asm,2021-08-12): This is sloppy, we should figure out a way to
-# get Python's native logging to behave the way we want.
-def verbose(msg):
-    print(f"[{datetime.now().isoformat()}] {msg}")
+    msg = generate_standard_error_msg(error_code=error_code, msg_args=msg_args)
+    if level == logging.WARNING or level == logging.WARN:
```

Comment:
Nit: WARN==WARNING internally, and logging.WARN has been deprecated since Python 3.3. Should remove WARN.